x-pack/plugins/security_solution/public/timelines/components/timeline/cell_rendering/default_cell_renderer.tsx

```diff
@@ -49,7 +50,7 @@ export const DefaultCellRenderer: React.FC<CellValueElementProps> = ({
     <>
       <StyledContent className={styledContentClassName} $isDetails={isDetails}>
         {getColumnRenderer(header.id, columnRenderers, data).renderColumn({
-          asPlainText: !!getLink(header.id, header.type), // we want to render value with links as plain text but keep other formatters like badge.
+          asPlainText: !!getLink(header.id, header.type) && !isTimeline, // we want to render value with links as plain text but keep other formatters like badge.
```

Comment:
consider adding the following tests to: `x-pack/plugins/security_solution/public/timelines/components/timeline/cell_rendering/default_cell_renderer.test.tsx
`

NOTE: the new `describe` must be added outside-of / below the existing `describe` block in `default_cell_renderer.test.tsx`, because it uses the actual implementation of `getColumnRenderer()` 

```typescript
describe('host link rendering', () => {
  const data = cloneDeep(mockTimelineData[0].data);
  const hostNameHeader = cloneDeep(defaultHeaders[4]);

  beforeEach(() => {
    const { getColumnRenderer: realGetColumnRenderer } = jest.requireActual(
      '../body/renderers/get_column_renderer'
    );

    getColumnRendererMock.mockImplementation(realGetColumnRenderer); // link rendering tests must use the real renderer
  });

  test('it renders a link button for `host.name` when `isTimeline` is true', () => {
    const id = 'host.name';
    const isTimeline = true;

    const wrapper = mount(
      <TestProviders>
        <DragDropContextWrapper browserFields={mockBrowserFields}>
          <DroppableWrapper droppableId="testing">
            <DefaultCellRenderer
              browserFields={undefined}
              columnId={id}
              ecsData={undefined}
              data={data}
              eventId="_id-123"
              header={hostNameHeader}
              isDetails={false}
              isDraggable={true}
              isExpandable={false}
              isExpanded={false}
              isTimeline={isTimeline}
              linkValues={[]}
              rowIndex={3}
              setCellProps={jest.fn()}
              timelineId={'timeline-1-query'}
            />
          </DroppableWrapper>
        </DragDropContextWrapper>
      </TestProviders>
    );

    expect(wrapper.find('[data-test-subj="host-details-button"]').first().text()).toEqual('apache');
  });

  test('it does NOT render a link button for `host.name` when `isTimeline` is false', () => {
    const id = 'host.name';
    const isTimeline = false;

    const wrapper = mount(
      <TestProviders>
        <DragDropContextWrapper browserFields={mockBrowserFields}>
          <DroppableWrapper droppableId="testing">
            <DefaultCellRenderer
              browserFields={undefined}
              columnId={id}
              ecsData={undefined}
              data={data}
              eventId="_id-123"
              header={hostNameHeader}
              isDetails={false}
              isDraggable={true}
              isExpandable={false}
              isExpanded={false}
              isTimeline={isTimeline}
              linkValues={[]}
              rowIndex={3}
              setCellProps={jest.fn()}
              timelineId={'timeline-1-query'}
            />
          </DroppableWrapper>
        </DragDropContextWrapper>
      </TestProviders>
    );

    expect(wrapper.find('[data-test-subj="host-details-button"]').exists()).toBe(false);
  });

  test('it does NOT render a link button for non-host fields when `isTimeline` is true', () => {
    const id = '@timestamp'; // a non-host field
    const isTimeline = true;
    const timestampHeader = cloneDeep(defaultHeaders[0]);

    const wrapper = mount(
      <TestProviders>
        <DragDropContextWrapper browserFields={mockBrowserFields}>
          <DroppableWrapper droppableId="testing">
            <DefaultCellRenderer
              browserFields={undefined}
              columnId={id}
              ecsData={undefined}
              data={data}
              eventId="_id-123"
              header={timestampHeader}
              isDetails={false}
              isDraggable={true}
              isExpandable={false}
              isExpanded={false}
              isTimeline={isTimeline}
              linkValues={[]}
              rowIndex={3}
              setCellProps={jest.fn()}
              timelineId={'timeline-1-query'}
            />
          </DroppableWrapper>
        </DragDropContextWrapper>
      </TestProviders>
    );

    expect(wrapper.find('[data-test-subj="host-details-button"]').exists()).toBe(false);
  });
});
```